src/RoleObjects/GridData.js

```diff
@@ -131,7 +131,8 @@ export default class GridData extends TableData {
     if (targetData.rowIndex < 0 && targetData.sectionIndex > 0) {
       // handles rolling into the prior table section
       targetData.sectionIndex--;
-      targetData.rowIndex = this.children[targetData.sectionIndex].accessible.children.length - 1; // eslint-disable-line max-len
+      targetData.rowIndex =
+        this.children[targetData.sectionIndex].accessible.children.length - 1; // eslint-disable-line max-len
```

Comment:
```suggestion
        this.children[targetData.sectionIndex].accessible.children.length - 1;
```
shouldn't need this exception now